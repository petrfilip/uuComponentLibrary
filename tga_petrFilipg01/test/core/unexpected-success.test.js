import UU5 from "uu5g04";
import TgaPetrFilip from "tga_petrFilipg01";

const { shallow } = UU5.Test.Tools;

describe(`TgaPetrFilip.Core.UnexpectedSuccess`, () => {
  it(`default props`, () => {
    const wrapper = shallow(<TgaPetrFilip.Core.UnexpectedSuccess />);
    expect(wrapper).toMatchSnapshot();
  });
});
