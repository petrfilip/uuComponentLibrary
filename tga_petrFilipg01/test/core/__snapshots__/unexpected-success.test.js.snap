// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TgaPetrFilip.Core.UnexpectedSuccess default props 1`] = `
<Component
  className="tgapetrfilip-core-0"
  style="paddingBottom: 10px"
>
  <Component>
    <Component
      className="center"
      colWidth="xs-12 s-12"
    >
      <Component>
        Mission accomplished?
      </Component>
    </Component>
  </Component>
  <Component>
    <Component
      className="center"
      colWidth="xs-12 s-6"
    >
      <Component
        colorSchema="success"
        onClick={[Function]}
      >
        Yes!
      </Component>
    </Component>
    <Component
      className="center"
      colWidth="xs-12 s-6"
    >
      <Component
        colorSchema="danger"
        onClick={[Function]}
      >
        No!
      </Component>
    </Component>
  </Component>
</Component>
`;
